{"name": "test_login[wrong_user-wrong_pass-False]", "status": "broken", "statusDetails": {"message": "AttributeError: 'LoginPage' object has no attribute 'open'", "trace": "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"2c3168a64ee79f1b4b436684152a68ad\")>, username = 'wrong_user', password = 'wrong_pass', is_valid = False\n\n    @pytest.mark.ui\n    @pytest.mark.parametrize(\"username,password,is_valid\", [\n        (\"standard_user\", \"secret_sauce\", True),\n        (\"locked_out_user\", \"secret_sauce\", False),\n        (\"wrong_user\", \"wrong_pass\", False),\n    ])\n    def test_login(driver, username, password, is_valid):\n        login_page = LoginPage(driver)\n>       login_page.open()\n        ^^^^^^^^^^^^^^^\nE       AttributeError: 'LoginPage' object has no attribute 'open'\n\ntests/ui/test_ui_login.py:13: AttributeError"}, "parameters": [{"name": "username", "value": "'wrong_user'"}, {"name": "password", "value": "'wrong_pass'"}, {"name": "is_valid", "value": "False"}], "start": 1749134373478, "stop": 1749134373478, "uuid": "afeb6224-1710-40da-a31e-12baa1a33dbe", "historyId": "1dd31125eeb64c72c1f03c32cec57760", "testCaseId": "4781fd2326e41bcc4eaaa21f5fb7f5a6", "fullName": "tests.ui.test_ui_login#test_login", "labels": [{"name": "tag", "value": "ui"}, {"name": "parentSuite", "value": "tests.ui"}, {"name": "suite", "value": "test_ui_login"}, {"name": "host", "value": "MacBook-Pro-Nikita.local"}, {"name": "thread", "value": "19262-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.ui.test_ui_login"}]}